src/stores/contextKeyStore.ts

```diff
@@ -0,0 +1,92 @@
+import { defineStore } from 'pinia'
+
+/**
+ * Store for context keys.
+ * Holds arbitrary key-value pairs globally throughout the app.
+ */
+export const useContextKeyStore = defineStore('contextKeys', {
+  state: () => ({
+    contextKeys: {} as Record<string, any>
```

Comment:
Please type the value of record here. The type of item we store in the pinia store should be concrete.